common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -93,6 +93,8 @@
 
 public abstract class BaseController {
 
+    // Declaring rawtypes warning name to avoid using the literal many times in this class
+    private static final String rawtype_warning = "rawtypes";
```

Comment:
I like the use of the constant; however can we use this everywhere?

Reply:
Sure @shoatman , I have now used the constant everywhere. Thanks.